docs/index.rst

```diff
@@ -1,39 +1,128 @@
 ################################
-Qiskit Experiments Documentation
+Qiskit Experiments documentation
 ################################
 
-About Qiskit Experiments
-========================
+.. warning::
+
+    This package is still under active development and it is very likely
+    that there will be breaking API changes in future releases.
+    If you encounter any bugs, please open an issue on
+    `Github <https://github.com/Qiskit/qiskit-experiments/issues>`_.
 
 Qiskit Experiments provides both a :doc:`library <apidocs/library>` of standard
 quantum characterization, calibration, and verification experiments, and a
 general :doc:`framework <apidocs/framework>` for implementing custom experiments
 which can be run on quantum devices through Qiskit.
 
-Experiments run on `IBMQ Provider <https://github.com/Qiskit/qiskit-ibmq-provider>`_
-backends can be stored and retrieved from an online experiment
-:doc:`database <apidocs/database_service>`.
+We've divided up the documentation into four sections with different purposes:
 
-.. warning::
+.. grid:: 2
+    :gutter: 5
+
+    .. grid-item-card::
+        :img-top: _static/tutorials.png
+
+        Tutorials
+        ^^^^^^^^^
+
+        These step-by-step tutorials teach the fundamentals of the package and 
+        are suitable for getting started.
+
+        You'll find in these tutorials:
+
+        * An overview of the package structure
+        * What constitutes an experiment
+        * How to write your own experiment
```

Comment:
I wonder if these bullet points should be links to the specific pages? What makes me suggest this is that I was looking at this first page and wondering what I would click on as a new user. Personally I would be trying to get to the getting started page (assuming that I came to the page knowing enough about qiskit-experiments to know that I wanted to use it). I think it would be nice if you could get from the initial page to the getting started page in one click instead of two.

Reply:
This is a good idea—I've added links.